### ISSUE TITLE
As a information planner I want a notification when stop name, route timetable, route or stops are changed

### ISSUE THREAD

**Issue by ossiberg** [HSLdevcom/jore4#172]
Possibility to link alerts to changelog (jore4#165) for a user?

Joona Packalen

**Comment by ellinooraikaheimo** [comment]
An email alert should be included in addition to jore4 notification? @ossiberg

**Comment by ossiberg** [comment]
I suppose a notification in Jore4 is enough for now, email notifications may be added later

**Comment by juhakolari** [comment]
Highlighted changes in the front pages of each section are planned and designed. Email notifications to be added later.
